app/controllers/quotes_controller.rb

```diff
@@ -1,26 +1,26 @@
 class QuotesController < ApplicationController
```

Comment:
The bulk of this file is the same, but my text editor re-indents the file using the rules I setup (spaces instead of tabs).

There were only 2 types of changes here:

Any instance of `Quotes` becomes -> `Quote`